### DISCUSSION TITLE
Smart Nora won't install for me

### DISCUSSION THREAD

**Discussion by MOOSEYCHOPS** [andrei-tatar/node-red-contrib-smartnora#268] [General]
I read the README, I have node red installed already through Venus os on RPi but I get the attached log when trying to install your smart node red. It doesn't seem to be a dependency (I might be wrong). I'm a real novice!

[log.odt](https://github.com/andrei-tatar/node-red-contrib-smartnora/files/12042479/log.odt)

**Comment by andrei-tatar** [comment]
@MOOSEYCHOPS from the log, it seems that the latest cbor version wants node >= 16 and you have node 14. 
I may have to downgrade cbor... Need to add a CI step to verify all dependencies for minimum node version.

**Reply by andrei-tatar** [reply]
@MOOSEYCHOPS I published version `v1.19.4`. Should work now with node 12+. Can you please check?
